modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/toolbars/SequenceDefaultToolbars.scala

```diff
@@ -273,12 +272,19 @@ object SequenceDefaultToolbar {
           SeqexecStyles.shorterRow,
           <.div(
             ^.cls := "ui left floated column eight wide computer eight wide tablet only",
-            p.controlReader(SequenceControl.apply)
+            p.controlReader(_() match {
+                case Some(c) => SequenceControl(SequenceControl.Props(c))
```

Comment:
This was avoided on the previous model where we would get the value without `Option`